eng/pipelines/templates/stages/archetype-java-release-batch.yml

```diff
@@ -0,0 +1,504 @@
+                    ArtifactDirectory: $(Pipeline.Workspace)/${{parameters.ArtifactName}}
+
+                - publish: $(Pipeline.Workspace)/${{parameters.ArtifactName}}
+                  artifact: ${{parameters.ArtifactName}}-signed
+                  displayName: 'Store signed packages in ${{parameters.ArtifactName}}-signed artifact'
+
+  # We generate two interdependent stages for each artifact listed in the ci.yml file, creates the release
+  # in GitHub. The Release stage publishes to Maven Central. Both stages require approval since they
+  # effectively burn the version number. For testing of packages prior to burning the version number -
+  # the Validation step below publishes a package to a "burner" feed which is cleaned up after the
+  # pipeline completes.
+  - ${{if and(in(variables['Build.Reason'], 'Manual', ''), eq(variables['System.TeamProject'], 'internal'))}}:
+    - stage:
+      variables:
+        numArtifacts: ${{ length(parameters.Artifacts) }}
+      displayName: 'Releasing: ${{ variables.numArtifacts }} libraries'
+      dependsOn: Signing
+      condition: and(succeeded(), ne(variables['SetDevVersion'], 'true'), ne(variables['Skip.Release'], 'true'), ne(variables['Build.Repository.Name'], 'Azure/azure-sdk-for-java-pr'))
+      jobs:
+        - job: VerifyReleaseVersion
+          displayName: "Verify release versions"
+          condition: ne(variables['Skip.VersionVerification'], 'true')
+          variables:
+            - template: /eng/pipelines/templates/variables/globals.yml
+          pool:
+            name: azsdk-pool-mms-ubuntu-2004-general
+            vmImage: MMSUbuntu20.04
+
+          steps:
+            - template: /eng/common/pipelines/templates/steps/sparse-checkout.yml
+              parameters:
+                Paths:
+                  - '${{ parameters.ServiceDirectory }}'
+                  - '**/*.xml'
+                  - '!sdk/**/test-recordings'
+                  - '!sdk/**/session-records'
+
+            - task: UsePythonVersion@0
+              displayName: 'Use Python 3.6'
+              inputs:
+                versionSpec: '3.6'
+
+            - template: /eng/common/pipelines/templates/steps/set-test-pipeline-version.yml
+              parameters:
+                PackageName: "azure-sdk-template"
+                ServiceDirectory: "template"
+                TestPipeline: ${{ parameters.TestPipeline }}
+
+            - template: /eng/common/pipelines/templates/steps/set-test-pipeline-version.yml
+              parameters:
+                PackageName: "azure-sdk-template-two"
+                ServiceDirectory: "template"
+                TestPipeline: ${{ parameters.TestPipeline }}
+
+            - template: /eng/common/pipelines/templates/steps/set-test-pipeline-version.yml
+              parameters:
+                PackageName: "azure-sdk-template-three"
+                ServiceDirectory: "template"
+                TestPipeline: ${{ parameters.TestPipeline }}
```

Comment:
We're doing set-test-pipeline-version for the 2 new template packages here.  Do we need to add set-test-pipeline-version for the 2 new packages over in archetype-java-release.yml?

Reply:
Good catch, I need to add those to archetype-java-release.yml as well. That got lost when I'd updated things for the "opt-in" approach. I'll get this updated in the next commit.